.github/workflows/actions.yml

```diff
@@ -10,8 +10,8 @@ on:
     - '**'
 jobs:
   build:
+    continue-on-error: true
```

Comment:
I think this is wrong and will result in commits marked as passing when they failed CI

Reply:
I'm pretty certain the overall status is still failed 🔴 (as on all recent PRs where truffleruby fails), its just that individual runs continue till the end.